backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandra.java

```diff
@@ -115,6 +116,18 @@ private void grantPermissionToTestingUser(Cluster privilegedCluster, String keys
         DockerfileBuilder applyStep(DockerfileBuilder builder);
     }
 
+    /**
+     * @return a string to append to image names in order to avoid conflict with concurrent builds
+     */
```

Comment:
I have a bit curious.
When concurrent builds?
Thank you

// The reason for this curiosity is my computer create a new docker image for each run integrate test.  (It takes up quite a bit of memory). So I want only one image.
Thank you

Reply:
I noticed the same. Maybe we should have an env variable dedicated to turn on that "unique C* docker image" on the CI.

What would you think about it @jeantil ?

Reply:
at the moment you have the opposite behavior where you can set BUILD_ID env variable and it will use a fixed image ... most CI platform I know use the `CI` env variable to let code know it is being run on CI, you could use that I guess.

Reply:
the reason the random extension was introduced was that we suspected a reaper process from deleting the images during the build using a random suffix made sure the image wasn't used for too long